lmdeploy/pytorch/engine/engine.py

```diff
@@ -300,18 +303,21 @@ def __update_max_new_tokens(msg):
             if len(sess.sequences) == 0:
                 assert len(
                     req.data['token_ids']) > 0, ('Empty input is not allowed.')
-                sess.add_sequence(req.data['token_ids'],
-                                  sampling_param=req.data['sampling_param'],
-                                  adapter_name=req.data['adapter_name'],
-                                  return_logits=req.data.get(
-                                      'return_logits', False))
+                sess.add_sequence(
+                    req.data['token_ids'],
+                    sampling_param=req.data['sampling_param'],
+                    adapter_name=req.data['adapter_name'],
+                    return_logits=req.data.get('return_logits', False),
+                    input_embeddings=req.data.get('input_embeddings'),
```

Comment:
Add a default value if input_embeddings has not been given.

Reply:
The default value is None by default.